examples/design_patterns/README.md

```diff
@@ -0,0 +1,75 @@
+# ML Infrastructure Design Patterns
+
+This directory describes a set of design patterns and annotated implementations for architecting machine learning
+systems with Runhouse based on common workflows. These patterns are based on the experience of
+both our team directly, our users' successes with Runhouse, and the architectures we've seen talking to hundreds of
+companies doing ML, large and small. They're designed to help you build reliable, scalable, cost-efficient, and
+maintainable ML systems.
+
+These patterns are designed to give you intuition and guide your thinking about how to utilize Runhouse.
+We've organized them based on the high-level workflows in which they're most likely to be found, but many of them
+are general-purpose and can be used in multiple contexts. We've also tried to include some context about what
+alternatives you might use today to achieve the same goals, and why you might choose one approach over another. We
+do not include example code for the alternatives, but we're happy to help you think through architectures and
+options in the [Runhouse discord](https://discord.gg/RnhB6589Hs).
+
+# 📚 Patterns
+
+We've organized the patterns into the following categories:
+    * [Optimizing I/O] - Strategies for synchronizing data communication with computation to minimize latency
```

Comment:
think you need an extra empty line here for bullet points to render properly

https://github.com/run-house/runhouse/blob/8c04029f3eae942f2094b98bd18aec2518ece2ea/examples/design_patterns/README.md